tensorbay/cli/utility.py

```diff
@@ -181,3 +178,93 @@ def _clean_up(editor_input: Optional[str]) -> Tuple[str, str]:
     if not cleaned_up_lines:
         return "", ""
     return cleaned_up_lines[0].lstrip(), "\n".join(cleaned_up_lines[1:])
+
+
+def update_config() -> None:
+    """Update the config file to the new format."""
+    old_config_parser = read_config()
+    if not old_config_parser.sections() or old_config_parser.sections() == ["config"]:
+        return
+
+    if (
+        old_config_parser.sections() == ["config", "profile"]
+        or old_config_parser.sections() == ["profile"]
+    ) and "accesskey" not in old_config_parser["profile"]:
+        return
+
+    new_config_parser = ConfigParser(dict_type=OrderedDict)
+
+    if old_config_parser.has_section("config"):
+        new_config_parser.add_section("config")
+        new_config_parser["config"] = old_config_parser["config"]
+
+    new_config_parser.add_section("profile")
+    for section in old_config_parser.sections():
```

Comment:
can we get section name and section from parser using method like `dict.items()`?